sapi/cli/tests/bug60471.phpt

```diff
@@ -0,0 +1,55 @@
+--TEST--
+Bug #60471 Random "Invalid request (unexpected EOF)" using a router script
+--SKIPIF--
+<?php
+include "skipif.inc";
+if (!PHP_DEBUG) die "skip preconnect messages only show in debug mode";
```

Comment:
`die` requires parens if it has an argument.

Reply:
![](https://media.giphy.com/media/nFn51VmBlX6W4/giphy.gif)

O. M. G. Sorry. Fixed. And sorry again. 🤦‍♂️